modules/migrations/github.go

```diff
@@ -535,6 +535,8 @@ func (g *GithubDownloaderV3) GetAllComments(page, perPage int) ([]*base.Comment,
 	if err != nil {
 		return nil, false, fmt.Errorf("error while listing repos: %v", err)
 	}
+	var isEnd = resp.NextPage == 0
+
 	log.Trace("Request get comments %d/%d, but in fact get %d", perPage, page, len(comments))
```

Comment:
It would be helpful to add the next page indicator to this TRACE statement

Reply:
done.